cgeglio/whats-new
# App state

#### What's this PR do?
- Dynamically adds menu buttons
- Adds a state to App that changes when a topic is selected
- Adds functionality to the menu: when a menu topic is clicked, the related news articles appear
- Improves styling and adds a logo
#### Where should the reviewer start?
- App.js
#### How should this be manually tested?
Click on the "entertainment" menu button and you should only see entertainment-related articles
